tests/results/DiffTest.py

```diff
@@ -60,31 +60,36 @@ def test_double_changes_with_same_diff(self):
             self.fail('We should not have a conflict on same diff!')
 
     def test_affected_code(self):
```

Comment:
To the same end, I'm now restructuring it so that one commit has all of the changes needed to make deprecated code still work, such that when the time comes a simple git revert will get rid of it